### DISCUSSION TITLE
Text Border?

### DISCUSSION THREAD

**Discussion by eric-hc** [tailwindlabs/tailwindcss#2768] [Help]
Is there a way to add a border or outline to text? Here's a rough example:

![Screen Shot 2020-11-15 at 4 28 07 PM](https://user-images.githubusercontent.com/1885157/99197285-9596da80-275f-11eb-9e57-d8b83cd55995.png)

I'd expect it to work something like `class="text-my-blue text-border-other-blue"`, but can't find an example or anything out there.

**Comment by erhankaradeniz** [comment]
I don't think it's currently possible with Tailwind, however you could use the text-shadow plugin to do this.. You might want to look at it: [https://github.com/iunteq/tailwindcss-textshadow#readme](https://github.com/iunteq/tailwindcss-textshadow#readme)
